### ISSUE TITLE
How to use configs

### ISSUE THREAD

**Issue by tethlah** [Jozufozu/ExNihiloOmnia#132]
I can't find any documentation on how to use the configs.  If I load into a world it doesn't create an island at all, it's just a void with nothing.  How do you use the configs to create the island in multiplayer?

**Comment by ghost** [comment]
you have only  ExNihiloOmnia to set the world geneation?
in Yes you can use \config\ExNihiloOmnia\ExNihiloOmnia.cfg to set the world generation

Example:
"world generation" {
    S:"overworld biome"=all
    B:"void end"=true
    B:"void end generate crystals"=true
    **S:"void end template"=none**
    B:"void nether"=true
    B:"void nether generate fortresses"=true
    **S:"void nether template"=none**
    B:"void overworld"=true
    **S:"void overworld template"=ex_nihilo_hard_overworld.json**
}

if you don't set a template yes it's create a void world, this applies both to the Nether End and Overworld

**Comment by tethlah** [comment]
Yes I have all that set, but ENO isn't doing anything.  All I did was make a superflat world with nothing but water at 200, set the templates to spawn at 200 (there is no bedrock), but it never makes an island anywhere when I spawn in.  I tried botania gardens of glass, but it always makes the islands at y1 for some reason.

**Comment by tethlah** [comment]
And before you say something about the water, I tried just letting the world generate, it just created a vanilla world with ground and all.  ENO didn't do anything to affect world generation.  Do you have to set a world type or something like with GOG?

**Comment by ghost** [comment]
I haven't set any world type, i left the default, and things seem to be going the right way, try to recreate a new world, as it regards the template can not tell you anything, i like hard setting with only tree , test with the default to see if things go

To create a Island try to see the TemplateExNihiloEasy and modify it

**Comment by tethlah** [comment]
Create an island????  Just use creative mode or something because that defeats the purpose?  Is there a command?  This is what I'm talking about, when I just leave the world as default and load in I load in a default world with rolling hills and rivers and stuff, ENO doesn't make any islands.
